src/backend/commands/tag.c

```diff
@@ -471,7 +471,8 @@ AddTagDescriptions(List *tags,
 		bool	isnull;
 		List	*allowed_values;
 		ListCell	*value_cell;
-		
+
+		value_cell = NULL;
```

Comment:
I prefer not to check the value of `value_cell` after the loop ends. It looks a black macro that the cell value is not necessarily null after the loop normally ends. It will be more robust to use a flag here.

Reply:
Okay, I followed your advice.